src/ui/Templates/CoreHelpers.ts

```diff
@@ -135,10 +136,10 @@ export interface ICoreHelpers {
    * Formats a numeric value using the format string.
    *
    * - `content`: The numeric value to format.
-   * - `format`: The format string to use. The options available are defined by
-   *   the [Globalize](https://github.com/klaaspieter/jquery-global#numbers) library.
+   * - `format`: Optional. The format string to use. The options available are defined by
+   *   the [Globalize](https://github.com/klaaspieter/jquery-global#numbers) library. Either a string or an object (see INumberFormatOptions).
    */
-  number: (content: string, format: string) => string;
+  number: (content: string, format: string | INumberFormatOptions) => string;
```

Comment:
Why do we need to support `INumberFormatOptions`?

Reply:
So you can pass the named helper option: `data-helper-options-format="n0"`